packages/connection/src/class-rest-connector.php

```diff
@@ -269,6 +269,7 @@ public function connection_reconnect() {
 				break;
 			case 'completed':
 				$response['status'] = 'completed';
+				do_action( 'jetpack_reconnection_completed' );
```

Comment:
Could you add a docblock here?

Reply:
Done, thanks a lot for pointing that out!